indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/ParallelIndexSupervisorTask.java

```diff
@@ -1817,6 +1824,25 @@ static Map<String, Object> getTaskReport(final OverlordClient overlordClient, fi
     }
   }
 
+  /**
+   * To be called only after the toolbox has been set in runTask
+   *
+   * @return the base input source with the toolbox
+   */
+  private InputSource getInputSource()
```

Comment:
Doesn't seem like this method is going to be used outside of `runTask`. I think it is better to just create and use the `inputSourceWithToolbox` inside `runTask`.

Reply:
Done